src/NodaTime.Web/Markdown/unstable/calendars.md

```diff
@@ -0,0 +1,223 @@
+
+The additional month in a leap year presents challenges for text handling, as well as for calendrical calculations in general.
+The support in Noda Time 1.3.0 should be seen as somewhat experimental, but feedback is very warmly welcomed. It's important
+to note that parsing and formatting of month names is *expected* to be incorrect in this version.
+
+Like the Islamic calendar, a Hebrew day properly starts at sunset, but this is not modelled within Noda Time.
+
+The `GetHebrewCalendar()` method accepts one parameter, specifying which month numbering system to use. The scriptural
+month numbering system uses Nisan as month 1, even though the new year (when the year number changes) occurs at the start of
+Tishri. In the scriptural system, Adar is month 12 in a non-leap year, and Adar I and Adar II are months 12 and 13 in a leap year.
+
+The civil month numbering system uses Tishri as month 1 (so the year number increases when the month number becomes 1 again,
+as in most calendars) but this means that Adar is month 6 in a non-leap year, and Adar I and Adar II are months 6 and 7 in a leap year.
+This then means that the subsequent months (Nisan, Iyar, Sivan, Tamuz, Av, Elul) have different numbers in leap and non-leap years.
+
+Unlike the parameters for the Islamic calendar, the month numbering in the Hebrew calendar doesn't affect any calculations - it *only*
+affects the numeric values of the months both accepted when constructing values (such as in the `LocalDate` constructor)
+and retrieving them (such as with `LocalDate.Month`).
+
+The convenience properties `HebrewScriptural` and `HebrewCivil` are just simpler alternatives to the `GetHebrewCalendar` method call.
+
+Wondrous
+===
+
+First supported in v2.3.0
+API access:
+
+- [`CalendarSystem.Wondrous`](noda-property://NodaTime.CalendarSystem.Wondrous)
+
+The [Wondrous calendar](https://en.wikipedia.org/wiki/Bah%C3%A1%27%C3%AD_calendar), also called the Bah�'� or Bad� calendar,
+is a solar calendar used by the Bah�'� Faith. A year consists of 19 months of 19 days, with 4 or 5 intercalary
+days between months 18 and 19. These intercalary days are called *Ayy�m-i-H�*, and are not regarded as belonging
+to any month. Until 2015, the length of Ayy�m-i-H� was related to the Gregorian calendar: 5 in a Gregorian leap
+year, and 4 otherwise. From 2015 onwards, the length of Ayy�m-i-H� is set to ensure that the new year (Naw-R�z)
+coincides with the vernal equinox in Tehran.
+
+The Noda Time implementation of the Wondrous calendar has two aspects which diverge from the
+"correct" calendar:
+
+- Days in the correct Wondrous calendar begin at sunset (so that is when the date changes);
+  in Noda Time the date always changes at midnight.
+- In order to fit in with the rest of the code base, Ayy�m-i-H� is modeled as extra days within
+  month 18. This not affects the reporting of month numbers and lengths, but additionally operations
```

Comment:
additionally --> additional

Reply:
Thanks, will fix.

Reply:
Actually, the "additionally" is correct - it's in addition to the previous comment rather than the operations themselves being additional. I've added a comma to make that clearer.